docs/quickstart.md

```diff
@@ -61,34 +61,15 @@
 
 ## Usage
 
-Once you generated the Prisma Client Go client and set up a datasource with Prisma, you're good to go!
-
-### Create the client and connect to the prisma engine
-
-```go
-client := db.NewClient()
-err := client.Connect()
-if err != nil {
-    handle(err)
-}
-
-defer func() {
-    err := client.Disconnect()
-    if err != nil {
-        panic(fmt.Errorf("could not disconnect %w", err))
-    }
-}()
-```
-
-### Full example
+Create a file `main.go`:
```

Comment:
In your `main.go`:

Reply:
And above, In your `schema.prisma`:

Reply:
I'm not sure if that's necessary, since most people probably use an editor and don't create the file via `touch`. And even if you use the command line, I personally just `vim x` and don't touch the file before writing to it. WDYT?

Reply:
Sounds good 👍